core/Azure.Mcp.Core/src/Areas/Server/Commands/ServiceCollectionExtensions.cs

```diff
@@ -118,6 +118,21 @@ public static IServiceCollection AddAzureMcpServer(this IServiceCollection servi
                     sp.GetRequiredService<ServerToolLoader>(),
                 };
 
+                // Always add utility commands (subscription, group) in namespace mode
+                // so they are available regardless of which namespaces are loaded
+                var utilityToolLoaderOptions = new ToolLoaderOptions(
+                    Namespace: ["subscription", "group"],
```

Comment:
Seems like we can refactor this so we're not having to hard code strings in the method when we have it in other places.